services/filemanager/fileManager_test.go

```diff
@@ -281,15 +281,15 @@ func TestFileManager(t *testing.T) {
 			for _, file := range fileList {
```

Comment:
It is also possible to write tests for Context, just passed a ctx that is already canceled, and expect it to fail. A very small timeout would have a similar effect for integration tests.